### PR TITLE
docs: Update the Landing Page Figma example to be interactive

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**
Update to the Figma example to be interactive - it was previously a static image.

Export-1727295877703

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: site/docs/pages/index.mdx [hunk 1]
```diff
@@ -528,7 +528,14 @@ import { FrameMetadata } from '@coinbase/onchainkit/frame';
 
 <section className="css-alternate-container w-full flex items-center justify-center">
   <aside className="flex flex-col pb-10 md:flex-row md:px-2 w-[940px]">
-    <img alt="OnchainKit figma" src="/assets/onchain-figma.png" width="924" loading="lazy" />
+    <iframe 
+      style={{border: '1px solid rgba(0, 0, 0, 0.1)'}}
```
Comment (alessey) [inline site/docs/pages/index.mdx:532]: can you add this as a class?

File: site/docs/pages/index.mdx [hunk 2]
```diff
@@ -528,7 +528,14 @@ import { FrameMetadata } from '@coinbase/onchainkit/frame';
...
+      width="924"
+      height="520"
```
Comment (alessey) [inline site/docs/pages/index.mdx:534]: base this on screensize?

File: site/docs/pages/index.mdx (hunk 1 above)
Comment (cpcramer) [reply site/docs/pages/index.mdx:532]: ahh whoops

File: site/docs/pages/index.mdx (hunk 2 above)
Comment (cpcramer) [reply site/docs/pages/index.mdx:534]: It actually resizes very well already

Comment (cpcramer) [reply site/docs/pages/index.mdx:534]: mobile:
Screenshot 2024-09-25 at 2 12 41 PM

Ipad:
Screenshot 2024-09-25 at 2 12 59 PM

File: site/docs/pages/index.mdx (hunk 1 above)
Comment (cpcramer) [reply site/docs/pages/index.mdx:532]: updated

**Review by alessey — APPROVED** [review]

**Review by 0xAlec — APPROVED** [review]

**Pull request merged**